src/components/flowchart-wrapper/flowchart-wrapper.js

```diff
@@ -35,63 +35,158 @@ const linkToFlowchartInitialVal = {
   showGoBackBtn: false,
 };
 
+const getKeyByValue = (object, value) => {
+  return Object.keys(object).find((key) => object[key] === value);
+};
+
 /**
  * Main flowchart container. Handles showing/hiding the sidebar nav for flowchart view,
  * the rendering of the flowchart, as well as the display of all related modals.
  */
 export const FlowChartWrapper = ({
-  flags,
   fullNodeNames,
+  graph,
   loading,
+  metadataVisible,
   modularPipelinesTree,
   nodes,
-  onLoadNodeData,
   onToggleFocusMode,
   onToggleModularPipelineActive,
   onToggleModularPipelineExpanded,
-  onUpdateActivePipeline,
+  onToggleNodeSelected,
   pipelines,
   sidebarVisible,
-  metadataVisible,
 }) => {
   const history = useHistory();
+  const { pathname, search } = useLocation();
+  const searchParams = new URLSearchParams(search);
 
-  // Reload state is to ensure it will call redirectLocation
-  // only when the page is reloaded.
-  const [reload, setReload] = useState(false);
+  const [errorMessage, setErrorMessage] = useState({});
+  const [invalidUrl, setInvalidUrl] = useState(false);
```

Comment:
Thinking we can use our usual `is` convention for boolean state values.

```suggestion
  const [isInvalidUrl, setIsInvalidUrl] = useState(false);
```